pkg/sysinfo/sysinfo_linux.go

```diff
@@ -33,6 +33,11 @@ func New(quiet bool) *SysInfo {
 		if !sysInfo.OomKillDisable && !quiet {
 			logrus.Warnf("Your kernel does not support oom control.")
 		}
+		_, err = ioutil.ReadFile(path.Join(cgroupMemoryMountpoint, "memory.swappiness"))
```

Comment:
Really not sure about all these warnings.
mem limit makes sense because it is pretty fundamental to resource control and somewhat important to know at daemon startup.